aict_tools/configuration.py

```diff
@@ -201,48 +212,156 @@ def __init__(self, config):
+            )
+
+    def parse_cta(self, config):
+        model_config = config['disp']
+        SIMPLE_OPTIONS = set([
+            'coordinate_transformation',
+            'source_az_column',
+            'source_alt_column',
+            'pointing_az_column',
+            'pointing_alt_column',
+            'source_az_unit',
+            'source_alt_unit',
+            'pointing_az_unit',
+            'pointing_alt_unit',
+            'focal_length_column',
+            'focal_length_unit',
+            'cog_x_column',
+            'cog_y_column',
+            'delta_column',
+            'delta_unit',
+            'focal_length_column',
+            'focal_length_unit',
+        ])
+        if SIMPLE_OPTIONS.intersection(model_config.keys()):
+            raise TypeError(
+                'You are trying to use configuration keys for the simple '
+                'data format while setting the data_format to "CTA". '
+                'There should be no reason to do this as the DL1-format is fixed'
+            )
+
+        self.output_name = model_config.get('output_name', 'disp_predictions')
+        self.coordinate_transformation = 'CTA'
+        self.source_az_column = 'true_az'
+        self.source_alt_column = 'true_alt'
+        self.source_zd_column = None
+        self.pointing_az_column = 'azimuth'
+        self.pointing_alt_column = 'altitude'
+        self.pointing_zd_column = None
+        self.focal_length_column = 'equivalent_focal_length'
+        if config['datamodel_version'] > '1.1.0':
+            self.cog_x_column = 'hillas_fov_lon'
+            self.cog_y_column = 'hillas_fov_lat'
+        else:
+            self.cog_x_column = 'hillas_x'
+            self.cog_y_column = 'hillas_y'
+        self.delta_column = 'hillas_psi'
+
+        for coord in ('alt', 'az', 'zd'):
+            col = f'source_{coord}_unit'
+            setattr(self, col, u.Unit('deg'))
+            for coord in ('alt', 'az', 'zd'):
+                col = f'pointing_{coord}_unit'
+                setattr(self, col, u.Unit('rad'))
+        self.delta_unit = u.Unit('deg')
+        self.focal_length_unit = u.Unit('m')
+
+        cols = {
+            self.cog_x_column,
+            self.cog_y_column,
+            self.delta_column,
+        }
+
+        cols.update(model_config['features'])
+        if self.feature_generation:
+            cols.update(self.feature_generation.needed_columns)
+        # Add id's because we generate new tables instead of adding columns
+        # and want these to be included
+        # focal_length is necessary for coordinate transformations
+        cols.update(['tel_id', 'event_id', 'obs_id', 'equivalent_focal_length'])
+        self.columns_to_read_apply = list(cols)
+        cols.update({
+            self.pointing_az_column,
+            self.pointing_alt_column,
+            self.source_az_column,
+            self.source_alt_column,
+        })
+
+        # size relevant here?
```

Comment:
Yes, it should be `hillas_intensity`. We have some performance plots  vs. size, so for that it is needed.